components/rmrk/Create/CreateCollection.vue

```diff
@@ -38,15 +38,24 @@
       </template>
 
       <template #footer>
-        <NeoField
-          v-if="isLogIn"
-          variant="danger"
-          :message="balanceNotEnoughMessage">
+        <NeoField v-if="isLogIn">
           <SubmitButton
             expanded
-            label="create collection"
+            :label="balanceNotEnough ? 'not enough funds' : 'create collection'"
```

Comment:
i18n translation

Reply:
The translation are defined here 
https://github.com/pbkompasz/nft-gallery/blob/21ef9ebd4bb83591215fe10da85473ca0264a505/locales/en.json#L38